pkg/auth/routes/handler.go

```diff
@@ -0,0 +1,94 @@
+package routes
+
+import (
+	"context"
+	"encoding/json"
+	"monorepa/model"
+	"monorepa/service/auth"
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+type HandlerItemsServ struct {
+	router   *mux.Router
+	services *auth.MockAuthService
+	ctx      context.Context
+}
+
+func New(ctx context.Context, router *mux.Router) *HandlerItemsServ {
+	return &HandlerItemsServ{
+		router:   router,
+		services: &auth.MockAuthService{},
+		ctx:      ctx,
+	}
+}
+
+func (h *HandlerItemsServ) HandlerItems() {
+	h.router.HandleFunc("/login", h.GetJWTToken).Methods("GET")
+	h.router.HandleFunc("/get-cert/{version}", h.GetCertKey).Methods("GET")
```

Comment:
better use "certificate" instead of get-cert

Reply:
Oleksii requirements:
Auth service
/ login { UserName: string Password  (not empty) } -> JWT token {UserName, KeyVersion}
/ get-cert/{version}